src/styles/themes/light.js

```diff
@@ -87,6 +87,7 @@ lightTheme.PAGE_BACKGROUND_COLORS = {
     [SCREENS.HOME]: lightTheme.sidebar,
     [SCREENS.SETTINGS.PREFERENCES]: colors.blue500,
     [SCREENS.SETTINGS.WORKSPACES]: colors.pink800,
+    [ROUTES.SETTINGS_STATUS]: colors.green700,
```

Comment:
Heads-up, the keys here need to be screen names, not routes. I will fix this in https://github.com/Expensify/App/pull/27482

Reply:
Thanks you, I was following the pattern with the other routes, so I left a review about adding documentation for the pattern you introduced